src/e2e-test/features/aribasource/AribaRunTime.feature

```diff
@@ -31,7 +31,7 @@ Feature: Ariba Source - Run time scenarios
     And Enter input plugin property: "clientId" with value: "admin.clientid" for Credentials and Authorization related fields
     And Enter input plugin property: "clientSecret" with value: "admin.clientsecret" for Credentials and Authorization related fields
     And Enter input plugin property: "apiKey" with value: "admin.apiKey" for Credentials and Authorization related fields
-    And Wait with a timeout of 60000 seconds
+    And Wait with a timeout of 60000 milli seconds
```

Comment:
I understand you need a longer timeout for pipeline runs. However, you have added sleeps in other places like here - after entering a config property, doing validation. These extra sleeps seem unnecessary. Can we remove them?